drivers/gpio/gpio_cy8c95xx.c

```diff
@@ -287,7 +287,7 @@ static const struct cy8c95xx_config cy8c95xx_##idx##_cfg = { \
 	.common = { \
 		.port_pin_mask = 0xFF, \
 	}, \
-	.i2c = I2C_DT_SPEC_INST_GET(idx), \
+	.i2c = I2C_DT_SPEC_GET(DT_INST(0, cypress_cy8c95xx_gpio)), \
```

Comment:
Well but then you'd always get the device 0 address and can't support multiple devices with different addresses. How about `.i2c = I2C_DT_SPEC_GET(DT_PARENT(DT_INST(idx, DT_DRV_COMPAT))),`?

Reply:
Thanks for the review!